staging/kube-prometheus-stack/values.yaml

```diff
@@ -3533,7 +3605,7 @@ thanosRuler:
     image:
       registry: quay.io
       repository: thanos/thanos
-      tag: v0.30.1
+      tag: v0.30.2
```

Comment:
do we want to align this with `thanos` k-app to use `v0.31.0`?

Reply:
hmm I might just stick with the verified version that works with KPS. esp if it doesn't help us if it would still be a separate image going into the airgapped bundle.

(and instead of making the change here, we should override this in k-apps defaults if we want to use a different image)